packages/terra-show-hide/package.json

```diff
@@ -35,7 +35,8 @@
     "prop-types": "^15.5.8",
     "terra-mixins": "^1.40.0",
     "terra-theme-context": "^1.0.0",
-    "terra-toggle": "^3.56.0"
+    "terra-toggle": "^3.56.0",
+    "uuid": "7.0.3"
```

Comment:
We are using uuid `3.4.0` now

Reply:
Good catch! Fixed in https://github.com/cerner/terra-core/pull/3838/commits/aa41c4b79d15cae9e48fece9b1bcbdc462dbe454